### PR TITLE
feat(#261): add benchmark for istanbuljs

### PR DESCRIPTION
**Pull request by ssibrahimbas**
#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

#### What Has New?

Added benchmark for IstanbulJS, a new NodeJS Framework

#### What Has Changed?

- Added `@istanbul/app` and `@istanbul/http` packages to `package.json`
- Added istanbul to `./lib/packages.js`
- Added `istanbul.js` to `./benchmarks/`

#### What Has Solved?

- #261 is fixed

### CODE REVIEW

File: lib/packages.js [hunk 1]
```diff
@@ -1,75 +1,76 @@
-'use strict'
+"use strict";
```
Comment (jsumners) [inline lib/packages.js:1]: Please do not change the formatting.

**Review by jsumners — DISMISSED** [review]

File: lib/packages.js (hunk 1 above)
Comment (ssibrahimbas) [reply lib/packages.js]: Okey, I'm fixed at b2b6cfa

File: benchmarks/istanbul.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+"use strict";
```
Comment (jsumners) [inline benchmarks/istanbul.js:1]: Please follow the project's code formatting. Run `npm run lint` locally.

Comment (ssibrahimbas) [reply benchmarks/istanbul.js]: okey, I'm fixed errors

**Review by RafaelGSS — APPROVED** [review]

**Pull request merged**